### PR TITLE
Comment-box (without delete)

### PR DESCRIPTION
**Pull request by SSalekin**

### CODE REVIEW

**Comment by SSalekin** [comment]
screenshot from 2016-03-30 09 59 24

screenshot from 2016-03-30 10 00 32

**Comment by MuhammadTamzid** [comment]
LGTM

**Comment by sujoydatta** [comment]
OK

File: public/scripts/commentbox.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+  },
+
+  render : function(){
+    return (
+      <div className="comment well">
+        <h3 className="commentAuthor">
+          {this.props.author}
+        </h3>
+        <span dangerouslySetInnerHTML={this.rawMarkup()}></span>
+        <button className="btn btn-xs btn-danger">Delete</button>
```
Comment (dinhhoanglong91) [inline public/scripts/commentbox.js]: No delete button at first PR

**Comment by SSalekin** [comment]
Removed the 'Delete' button
screenshot from 2016-03-31 09 48 42

File: public/scripts/commentbox.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+    if(!text || !author){
+      alert("Name / Comment cannot be empty");
+      return;
+    }
+    this.props.onCommentSubmit({author: author, text: text});
+    this.setState({author: '', text: ''});
+  },
+
+  handleDelete: function(e) {
```
Comment (dinhhoanglong91) [inline public/scripts/commentbox.js]: please remove it

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**